src/map/mob.cpp

```diff
@@ -4811,6 +4812,18 @@ uint64 MobDatabase::parseBodyNode(const ryml::NodeRef& node) {
 		mob->status.amotion = cap_value(speed, battle_config.monster_max_aspd, 2000);
 	}
 
+	if (this->nodeExists(node, "ClientAttackMotion")) {
+		uint16 speed;
+
+		if (!this->asUInt16(node, "ClientAttackMotion", speed))
+			return 0;
+
+		mob->status.clientamotion = cap_value(speed, 1, USHRT_MAX);
```

Comment:
We should not use cap value here.
Give a proper warning for 0 or > 432 (or newly defined constant).

Reply:
Those values do exist in the files. The reason I cap 0 to 1 is simply to prevent the risk of division by 0.
Values above 432 are valid.